src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -83,11 +81,11 @@ class ElectronicDeliveryForm extends React.Component {
         id="edd-request"
       >
         <fieldset>
```

Comment:
@ktp242 you updated the DOM but did not add the appropriate classes. Check out https://nypl.github.io/design-toolkit/edd-request-form.html and update the classnames. This should have `nypl-fieldset` and it will remove the border.

Reply:
OK Thanks for it! I will update it.